jetson/odrive_bridge/src/__main__.py

```diff
@@ -50,23 +50,30 @@ def main():
     # starting state is DisconnectedState()
     # start up sequence is called, disconnected-->disarm-->arm
 
+    prev_comms = False
+
     while True:
         watchdog = t.clock() - start_time

```

Comment:
Can you add a comment explaining what the watchdog variable is? It's a bit difficult to understand what it is being used for